cylc/flow/scheduler.py

```diff
@@ -2124,18 +2158,33 @@ def resume_workflow(self, quiet: bool = False) -> None:
         self.workflow_db_mgr.put_workflow_paused(False)
         self.update_data_store()
 
-    def command_force_trigger_tasks(self, items, flow, flow_wait, flow_descr):
+    def command_force_trigger_tasks(
+        self,
+        tasks: Iterable[str],
+        flow: List[str],
+        flow_wait: bool = False,
+        flow_descr: Optional[str] = None
+    ):
         """Manual task trigger."""
         return self.pool.force_trigger_tasks(
-            items, flow, flow_wait, flow_descr)
+            tasks, flow, flow_wait, flow_descr)
 
-    def command_force_spawn_children(self, items, outputs, flow_num):
+    def command_set(
+        self,
+        tasks: List[str],
+        flow: List[str],
+        outputs: Optional[List[str]] = None,
+        prerequisites: Optional[List[str]] = None,
+        flow_wait: bool = False,
+        flow_descr: Optional[str] = None
+    ):
         """Force spawn task successors.
 
-        User-facing method name: set_outputs.
-
+        User-facing method name: reset.
```

Comment:
"reset"?
```suggestion
```

Reply:
> I think task_globs still applies

That was a necessary change, queried by @oliver-sanders earlier: https://github.com/cylc/cylc-flow/pull/5658#discussion_r1339733156

Reply:
Ah ok

> What's happened here is that Hillary has removed an intermediate function that was converting the argument names from the GraphQL schema to the argument names used in these functions.
> 
> As a result the argument must now be given the same name as used in the GraphQL schema, which is `task` rather than the more informative `task_glob`.